ehr_ml/clmbr/__init__.py

```diff
@@ -555,3 +556,35 @@ def debug_model() -> None:
 
                 for a in items:
                     print(a)
+
+def convert_patient_data(extract_dir: str, original_patient_ids: List[int], date_strs: List[str]) -> Tuple[np.array, np.array]:
```

Comment:
Type signature is wrong here. Should be original_patient_ids: Iterable[int], date_strs: Iterable[str]